modules/installation-generate-aws-user-infra-install-config.adoc

```diff
@@ -109,6 +109,14 @@ imageContentSources:
 ----
 +
 Use the `imageContentSources` section from the output of the command to mirror the repository or the values that you used when you mirrored the content from the media that you brought into your restricted network.
+
+.. Optional: Set the publishing strategy to `Internal`
++
+----
```

Comment:
Would adding `[source,yaml]` to this (and any other relevant code blocks in this module) be appropriate?